.github/workflows/build-sample-app-for-sdk-release.yml

```
@@ -0,0 +1,17 @@
+name: Build sample app for SDK release
+on:
+  workflow_dispatch:
+jobs:
+  build-sample-apps:
+    uses: ./.github/workflows/reusable_build_sample_apps.yml
+    with:
+      use_latest_sdk_version: true
+    secrets:
+      CUSTOMERIO_JAVA_WORKSPACE_CDP_API_KEY: ${{ secrets.CUSTOMERIO_JAVA_WORKSPACE_CDP_API_KEY }}
+      CUSTOMERIO_JAVA_WORKSPACE_SITE_ID: ${{ secrets.CUSTOMERIO_JAVA_WORKSPACE_SITE_ID }}
+      CUSTOMERIO_KOTLIN_WORKSPACE_CDP_API_KEY: ${{ secrets.CUSTOMERIO_KOTLIN_WORKSPACE_CDP_API_KEY }}
+      CUSTOMERIO_KOTLIN_WORKSPACE_SITE_ID: ${{ secrets.CUSTOMERIO_KOTLIN_WORKSPACE_SITE_ID }}
+      ANDROID_SIGNING_ALIAS: ${{ secrets.ANDROID_SIGNING_ALIAS }}
+      ANDROID_SIGNING_KEY_PASSWORD: ${{ secrets.ANDROID_SIGNING_KEY_PASSWORD }}
+      ANDROID_SIGNING_STORE_PASSWORD: ${{ secrets.ANDROID_SIGNING_STORE_PASSWORD }}
+      FIREBASE_APP_DISTRIBUTION_SERVICE_ACCOUNT_CREDS_B64: ${{ secrets.FIREBASE_APP_DISTRIBUTION_SERVICE_ACCOUNT_CREDS_B64 }}
```

```suggestion
    uses: ./.github/workflows/reusable_build_sample_apps.yml
    secrets: inherit
```

we can probably simplify it by [inherting](https://docs.github.com/en/actions/sharing-automations/reusing-workflows#using-inputs-and-secrets-in-a-reusable-workflow) the secrets?

---

Great suggestion! Will do.